README.md

```diff
@@ -845,6 +845,11 @@ message on the console.
 The server SHOULD implement a guard time after which the server should accept a
 new module-start sequence.
 
+> Note: The implementation itself can choose between to measurement stop conditions:
+> a) Mission time: send Measurement Stop Request after a defined interval (i.e. 60 seconds).
+> b) Byte count: send Measurement Stop Request after a defined number of bytes. 
+> Our Mapago implementation currently supports the misison time as a stop condition. The condition "byte count" is WIP.
```

Comment:
Nope, there is no "our" in the standard. And that the reference implementation is implementing some features (thus WIP) makes also no sense in a standard. Who will update the document later?